source/lib/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@
 
 set (libname ${LIB_DEEPMD})
 
+# Devices that have both ROCM and CUDA are not currently supported
+if (USE_ROCM_TOOLKIT AND USE_CUDA_TOOLKIT)
+  message (FATAL_ERROR "Devices that have both ROCM and CUDA are not currently supported")
+endif()
+
```

Comment:
We have already died in source/CMakeList.txt. Do we really need to die again here?

Reply:
Because the TensorFlow version is one of rocm and cuda, so when faced with a choice between CUDA and ROCM, I threw out an error